app/models/factory.rb

```diff
@@ -107,4 +76,30 @@ def family
   def to_s
     name + ":" + traits.map(&:name).join(",").to_s
   end
+
+  private
```

Comment:
Lint/UselessAccessModifier: Useless `private` access modifier.<br>Style/EmptyLinesAroundAccessModifier: Keep a blank line before and after `private`.